docs/dbt/sources.mdx

```diff
@@ -47,16 +47,22 @@ The contents of the `mage_sources.yml` could look like this:
 
 ```yaml
 sources:
-  - name: mage_demo_project
-    tables:
-      - name: demo_pipeline_load_data
+- loader: mage
+  name: mage_demo
+  schema: public
+  tables:
+  - identifier: mage_dbt_demo_pipeline_load_data
+    meta:
+      block_uuid: load_data
+      pipeline_uuid: dbt_demo_pipeline
+    name: dbt_demo_pipeline_load_data
 version: 2
 ```
 
-The source name follows this convention: `mage_[Mage_project_name]`.
+The source name follows this convention: `mage_[pipeline_uuid]`.
```

Comment:
nit: this should be changed back as well `mage_[dbt_project_name]`
And the table name below should be changed back.